vrep_interface_tests/line_to_line_angle_jacobian/line_to_line_angle_jacobian.py

```diff
@@ -0,0 +1,166 @@
+Contributors:
+- Juan Jose Quiroz Omana (juanjqo@g.ecc.u-tokyo.ac.jp)
+Instructions:
+Prerequisites:
+- dqrobotics
+- dqrobotics-vrep-interface
+- quadprog
+
+1) Open the CoppeliaSim scene test_dynamic_conic_constraint.ttt
+from cpp-examples/cmake/vrep_interface_tests/line_to_line_angle_jacobian/
+'''
+
+from dqrobotics import *
+from dqrobotics.interfaces.vrep  import DQ_VrepInterface
+from dqrobotics.robot_control import ControlObjective
+from dqrobotics.robot_control import DQ_ClassicQPController
+from dqrobotics.robot_modeling import DQ_Kinematics, DQ_SerialManipulatorMDH
+from dqrobotics.solvers import DQ_QuadprogSolver
+from dqrobotics.utils import DQ_Geometry
+import time
+import numpy as np
+from math import pi, sin, cos
+
+
+vi = DQ_VrepInterface()
+
+USE_RESIDUAL = True  
+
+## Always use a try-catch in case the connection with V-REP is lost
+## otherwise your clientid will be locked for future use
+try:
+    ## Connects to the localhost in port 19997 with timeout 100ms and 10 retries for each method call
+    vi.connect(19997, 100, 10)
+    vi.set_synchronous(True)
+
+    ## Starts simulation in CoppeliaSim
+    print("Starting CoppeliaSim simulation...")
+    vi.start_simulation()
+    time.sleep(0.1)
+
+    iterations = 15000
+    jointnames = ("Franka_joint1", "Franka_joint2", "Franka_joint3", "Franka_joint4",
+                   "Franka_joint5", "Franka_joint6", "Franka_joint7")
+
+    #--------------------- Robot definition
+    franka_mdh = np.array([[0,    0,     0,         0,      0,       0,      0],
+                           [0.333, 0, 3.16e-1,       0, 3.84e-1,     0,      0],
+                           [0,     0,     0,   8.25e-2, -8.25e-2,    0, 8.8e-2],
+                           [0, -pi/2,   pi/2,     pi/2,    -pi/2,   pi/2, pi/2],
+                           [0,     0,     0,         0,        0,     0,    0]])
+    franka = DQ_SerialManipulatorMDH(franka_mdh)
+    robot_base = 1 + E_ * 0.5 * DQ([0, 0.0413, 0, 0])
+    franka.set_base_frame(robot_base)
+    franka.set_reference_frame(robot_base)
+    robot_effector = 1+E_*0.5*k_*1.07e-1
+    franka.set_effector(robot_effector)
+
+    # Update the base of the robot from CoppeliaSim
+    new_base_robot = DQ([1])
+    for i in range(10):
+        new_base_robot = (franka.get_base_frame())*vi.get_object_pose("Franka")*(1+0.5*E_*(-0.07*k_))
+
+    franka.set_base_frame(new_base_robot)
+    franka.set_reference_frame(new_base_robot)
+
+    #------------------- Controller definition---------------------
+    solver = DQ_QuadprogSolver()
+    controller = DQ_ClassicQPController(franka, solver)
+    controller.set_gain(0.5)
+    controller.set_damping(0.1)
+    controller.set_control_objective(ControlObjective.Translation)
+    controller.set_stability_threshold(0.001)
+    #---------------------------------------------------------------
+    vfi_gain = 0.5
+    safe_angle = 15*(pi/180)
+    T = 0.005
+    w = 0.2
+    Alpha = 20*(pi/180)
```

Comment:
The uppercase `Alpha` is weird. Any special reason?

Reply:
@mmmarinho 
There is no special reason. I guess I was using the classic style definition of a sinusoid in which the amplitude is represented with an uppercase letter.

Source:  [Matlab documentation](https://www.mathworks.com/help/dsp/ref/dsp.sinewave-system-object.html)
![Screenshot 2022-11-17 105959](https://user-images.githubusercontent.com/23158313/202337890-9546b167-098c-46bb-84a1-b4ff91f23312.png)

However, to keep the convention of uppercase for matrices and lowercase for scalars, I modified the variable from `Alpha` to `alpha`.

Best regards, 

Juancho

Reply:
Sorry, @juanjqo, but I have to comment on this one. The classic definition uses the Latin letter `A` for the amplitude, not `Alpha`! 🤣

If this is a local variable that refers to the amplitude, then I suggest using `amplitude`.

Best,
Bruno

Reply:
@bvadorno you're right! 😂